scripts/nim/build-ckeditor5.mjs

```diff
@@ -42,7 +42,9 @@ function dist( path ) {
 		 * We don't want to repeat this in other steps.
 		 */
 		clean: true,
-		declarations: true,
+
+		// Temporarily disabled to avoid problems caused by duplicate declarations.
+		// declarations: true,
```

Comment:
```suggestion

		// Temporarily disabled to avoid problems caused by duplicate declarations.
		declarations: false,
```

Or, remove the entire line. We do not need a commented code if we can restore it from git history.